src/features/hotspots/details/HotspotDetails.tsx

```diff
@@ -317,6 +376,27 @@ const HotspotDetails = ({
             />
           </>
         )}
+
+        {selectedOption === 'witnesses' && (
+          <>
+            {hotspotDetailsData.loading ? (
+              <Box marginTop="xl">
+                <ActivityIndicator color="#81909F" />
+              </Box>
+            ) : (
+              <>
+                <Box backgroundColor="grayBox" marginBottom="xxs" marginTop="m">
+                  <Text variant="body1Medium" color="grayDarkText" padding="m">
+                    {t('hotspot_details.num_witnesses', {
+                      count: witnesses?.length || 0,
+                    })}
+                  </Text>
+                </Box>
+                {witnesses?.map((witness) => renderWitnessItem(witness))}
```

Comment:
I had this as a FlatList originally and it works, but since this is already inside a scroll view react throws an error. So I changed it to just directly render them in the scrollview. Open to ideas if you have any.

Reply:
The way you have it feels right to me.